Assets/MRTK/Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset

```diff
@@ -16,109 +16,109 @@ MonoBehaviour:
   useManagerScene: 0
   managerScene:
     Name: DefaultManagerScene
-    Path: Assets/MRTK/SDK/StandardAssets/Scenes/DefaultManagerScene.unity
-    Included: 0
+    Path: Assets/MRTK/SceneSystemResources/DefaultManagerScene.unity
+    Included: 1
     BuildIndex: -1
     Tag: Untagged
     Asset: {fileID: 102900000, guid: ae7bb08d297fb69408695d8de0962524, type: 3}
   useLightingScene: 0
   defaultLightingSceneIndex: 0
   lightingScenes:
   - Name: DefaultLightingScene
-    Path: Assets/MRTK/SDK/StandardAssets/Scenes/DefaultLightingScene.unity
-    Included: 0
+    Path: Assets/MRTK/SceneSystemResources/DefaultLightingScene.unity
+    Included: 1
     BuildIndex: -1
     Tag: Untagged
     Asset: {fileID: 102900000, guid: 7e54e36c44f826c438c95da79f8de638, type: 3}
   contentScenes:
   - Name: MRTKExamplesHubMainMenu
     Path: Assets/MRTK/Examples/Experimental/ExamplesHub/Scenes/MRTKExamplesHubMainMenu.unity
-    Included: 0
-    BuildIndex: -1
+    Included: 1
```

Comment:
I think you may want to revert some of these, since they aren't actually included in this repo's scene list.

Reply:
all of the profiles or just this one? the reason i kept the changes was due to these being immutable when installed

Reply:
oh! just the Included: 1, i presume

Reply:
and the build index....

Reply:
fixed